### PR TITLE
Fancy underline behind text

### PR DESCRIPTION
**Pull request by lpil**
# oops.

I can't make this work cross browser with multiple font sizes and multiple lines. Shelving this for now.

---

### Motivation
We wanna look reet good.

### Test plan
- [x] View hoverstate of desktop header nav links on supported browsers
- [x] View desktop footer nav links on supported browsers

### Pre-merge checklist

- [x] Reviews
  - [x] Code review
  - [x] Design review
- [x] Manual testing
  - [x] Latest Chrome
  - [x] Latest Firefox
  - [x] IE11
  - [x] Edge
  - [x] Latest Safari
  - [x] iPhone 6
  - [x] Google Nexus 5
  - [x] Samsung S6
  - [x] iPad (5th gen)
- [ ] Tester approved

### CODE REVIEW

**Comment by asavin** [comment]
screen shot 2016-10-10 at 16 13 16

Mobile nav is broken

File: site/components/header/style.css [hunk 1]
```diff
@@ -132,3 +127,63 @@
...
+    .smallScreenNavLink:hover {
+      line-height: 1;
+      text-decoration: none;
+      background-image: linear-gradient( to right, red 0%, red 100% );
+      background-repeat: repeat-x;
+      background-position: 0 96%;
+      background-size: 100% 2px;
+      color: white;
+      text-shadow:
+         0.09em   0       0 black,
```
Comment (lpil) [inline site/components/header/style.css]: @asavin says "Make this a variable".

**Comment by AndrewBestbier** [comment]
Merge conflicts

**Comment by asavin** [comment]
There's regression for the mobile nav links - they should be all underlined by default with no need to hover

**Comment by lpil** [comment]
@AndreiDanielIonescu Ready for test :)

**Comment by AndreiDanielIonescu** [comment]
On windows, on all browsers , the underline for ''open in google maps'' gets corrupted visually while zooming.

**Pull request closed without merging**